java/src/main/resources/META-INF/native-image/com.github.luben/zstd-jni/index.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
Why isn't it same kind of metadata description as the other?

Reply:
Without it I had some issues at native image building phase.

Reply:
ok so this is a new way to define metadata